src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/__tests__/ExposureDetectionViewModelTests.swift

```diff
@@ -11,8 +11,31 @@ import OpenCombine
 // swiftlint:disable:next type_body_length
 class ExposureDetectionViewModelTests: XCTestCase {
 
+	private var otpService: OTPServiceProviding!
+	private var ppacToken: PPACToken!
+	private var store: MockTestStore!
+	private var client: Client!
+	private var riskExpectation: XCTestExpectation?
+	
+	override func setUp() {
+		super.setUp()
+		
+		self.store = MockTestStore()
+		self.client = ClientMock()
+		self.otpService = OTPService(store: store, client: client)
+		self.ppacToken = PPACToken(apiToken: "apiTokenFake", deviceToken: "deviceTokenFake")
+	}
+	
+	override func tearDown() {
+		otpService = nil
+		ppacToken = nil
+		client = nil
+		store = nil
+		
+		super.tearDown()
+	}
```

Comment:
I doubt we need this because we initialize new instances of these properties in `setUp()`.
```suggestion
```

Btw, did you know we can tear down inline within an `addTeardown()` block? I just found out :) https://developer.apple.com/documentation/xctest/xctestcase/understanding_setup_and_teardown_for_test_methods

Reply:
Removed it and nice find!